open-company/open-company-web
# Redirect to stream view on publish

To test:
- go to AP
- add a post
- publish it
- [x] you still in AP after publish?
- [x] see the newly added post?
- [x] is it read?
- go to Section A
- add a post to Section A
- publish it
- [x] you still in Section A after publish?
- [x] see the newly added post?
- [x] is it read?
- go to Section A 
- add a post to Section B
- publish it
- [x] are you redirected to Section B after publish?
- [x] see the newly added post?
- [x] is it read?
